gr-digital/grc/digital_gfsk_mod.block.yml

```diff
@@ -29,6 +29,13 @@ parameters:
     options: ['True', 'False']
     option_labels: ['On', 'Off']
     hide: ${ ('part' if str(log) == 'False' else 'none') }
+-   id: to_unpack
+    label: Unpack
+    dtype: bool
+    default: 'True'
+    options: ['True', 'False']
+    option_labels: ['On', 'Off']
+    hide: ${ ('part' if str(to_unpack) == 'False' else 'none') }
```

Comment:
`'part' if to_unpack else 'none'` should work.

Reply:
Oh I wasnt aware you want the gfsk mod block's look unchanged when the unpack feature is enabled. Modified as requested